doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -290,7 +295,27 @@ using two ``ml.p4d.24xlarge`` instances:
         }
     )
 
```

Comment:
can we add:
For more information about setting up PyTorch Torchrun in your training script,
see `TORCHRUN documentation
<https://pytorch.org/docs/stable/elastic/run.html>`_ in the
PyTorch documentation.

here?